.circleci/config.yml

```diff
@@ -193,6 +195,24 @@ jobs:
       - ruby/install-deps:
           key: gems-v2
       - ruby/rubocop-check
+  erblint:
+    executor: default
+    steps:
+      - checkout
+      - ruby/install-deps:
+          key: gems-v2
+      - run:
+          name: Lint ERBs
+          command: bundle exec erblint --lint-all --format compact
```

Comment:
While adding the JS linting, I discovered that we have an ERB linter we no longer use. Why? It looks like we missed it when switching from TravisCI to CircleCI, which I believe was probably not intended.